fluster/test.py

```diff
@@ -95,13 +95,28 @@ def _test(self) -> None:
 
         if not self.reference:
             self.test_suite.test_vectors[self.test_vector.name].test_result = TestVectorResult.FAIL
-            if self.test_vector.result.lower() == result.lower():
+            if self.test_suite.name in {"MPEG2_VIDEO-422", "MPEG2_VIDEO-MAIN"}:
```

Comment:
I don't think that's the correct way of implementing it since you are coupling the test method to a specific set of test suites by name.

Instead of implementing different testing methods in the same `Test` class there should one class for each test method, ending with a  `PixelComparisonTest` class and renaming the current one to `MD5ComparisonTest`.

You can either force `PixelComparisonTest` for all test suites using the MPEG2 codec or you can make it fully flexible by adding a `test_method` property to test suites where a test suite can configure the test method to use,  defaulting to the md5 one.

You would then create one test class or the other based on that here: https://github.com/fluendo/fluster/blob/master/fluster/test_suite.py#L552